query-engine/query-engine/src/opt.rs

```diff
@@ -11,11 +11,11 @@ pub enum Subcommand {
     Cli(CliOpt),
 }
 
-#[derive(Debug, Clone, StructOpt)]
-pub struct DmmfToDmlInput {
-    #[structopt(name = "path")]
-    pub path: String,
-}
+// #[derive(Debug, Clone, StructOpt)]
+// pub struct DmmfToDmlInput {
+//     #[structopt(name = "path")]
+//     pub path: String,
+// }
```

Comment:
Can this be deleted?

Reply:
Yes, I commented it out to see whether anything breaks and also confirmed with Tim that we do no longer go from dmmf-> dml